### DISCUSSION TITLE
My Found Problems(0.0.7 Linux X11)

### DISCUSSION THREAD

**Discussion by IHateGameDev** [SpaRcle-Studio/SREngine#261] [Show and tell]
### V0.1_Eng:
[myProblems_0.0.7_0.1_eng.md](https://github.com/user-attachments/files/16094444/myProblems_0.0.7_0.1_eng.md)
### V0.1_Ru:
[myProblems_0.0.7_0.1_ru.md](https://github.com/user-attachments/files/16094445/myProblems_0.0.7_0.1_ru.md)
### Logs:
**Log1:**
[srengine-log_1tr.txt](https://github.com/user-attachments/files/16094446/srengine-log_1tr.txt)
**Log2:**
[srengine-log_VulkansErrors.txt](https://github.com/user-attachments/files/16094447/srengine-log_VulkansErrors.txt)

**Comment by innerviewer** [comment]
> "Collision shape type": Plane3d - does not work. 

We know about this. For now, you can use Box3D with small values instead. 

> the Capsule3d parameters rolled back to the default value 

Are you sure you saved the scene correctly? 

> Convex3d, part of the collider fell under another Box3d collider

I think this is due to the way PhysX bakes the meshes. We are working on our own system for convex meshes.


> About Creation 
> About Cons
> About other issues

These problems are very specific and I am hearing them for the first time. Can you join our Discord server so we can discuss them? 
I'll make sure to post here afterwards so that anyone experiencing it can find the answer.
